pkg/network/event_common.go

```diff
@@ -18,6 +18,13 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 )
 
+const (
+	// 100Gbps * 30s = 375GB
+	maxByteCountChange uint64 = 375 << 30
+	// jumbo frame size (MTU) of 9216 = 9 * 1024
+	maxPacketCountChange uint64 = maxByteCountChange / (9 << 10)
```

Comment:
for _maximum_ packet count change, should we consider a smaller MTU?

Reply:
Good catch, yes this should be the smallest MTU. `1300` on AWS? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/network_mtu.html